frontend/src/metabase/admin/datamodel/components/revisions/RevisionHistory.jsx

```diff
@@ -51,9 +50,9 @@ export default class RevisionHistory extends Component {
                 {revisions.map(revision => (
                   <Revision
                     revision={revision}
-                    objectName={name}
+                    objectName={object.name}
                     currentUser={user}
-                    tableMetadata={tableMetadata}
+                    tableMetadata={table}
```

Comment:
@tlrobinson, I renamed a bunch of `tableMetadata` to `table` (same with database). This was done to stay consistent the entity loader naming. However, I wasn't confident enough that this is the "new" naming convention to propagate it down through all the components. Should I?